x-pack/plugin/core/src/javaRestTest/java/org/elasticsearch/xpack/core/DataStreamRestIT.java

```diff
@@ -42,19 +43,24 @@ public void testDSXpackInfo() {
         assertTrue((boolean) dataStreams.get("enabled"));
     }
 
+    @SuppressWarnings("unchecked")
     public void testDSXpackUsage() throws Exception {
         Map<?, ?> dataStreams = (Map<?, ?>) getLocation("/_xpack/usage").get("data_streams");
         assertNotNull(dataStreams);
         assertTrue((boolean) dataStreams.get("available"));
         assertTrue((boolean) dataStreams.get("enabled"));
         assertThat(dataStreams.get("data_streams"), anyOf(equalTo(null), equalTo(0)));
-
+        assertThat(dataStreams, hasKey("failure_store"));
+        Map<String, Integer> failureStoreStats = (Map<String, Integer>) dataStreams.get("failure_store");
+        assertThat(failureStoreStats.get("enabled"), anyOf(equalTo(null), equalTo(0)));
```

Comment:
```suggestion
        assertThat(failureStoreStats.get("enabled_count"), anyOf(equalTo(null), equalTo(0)));
```
The CI didn't fail, so adding `hasKey` checks might be valuable here?

Reply:
Where was my mind when I was writing this oops....